libs/core/src/components/sage-checkbox/sage-checkbox.scss

```diff
@@ -90,36 +120,6 @@ label {
   width: 100%;
 }
 
-.sage-checkbox--indeterminate {
-  input:checked {
-    &::after {
-      border-bottom: 0;
-      transform: rotate(87deg) translate3d(-114%, 25%, 0);
-    }
-  }
-}
-
-.sage-checkbox--invalid {
-  input {
-    border-color: var(--invalid-color);
-
-    &:checked {
-      background: var(--invalid-color);
-      border-color: var(--invalid-color);
-    }
-
-    &:focus-visible {
-      box-shadow: 0 0 0 4px var(--invalid-focus-color);
-      outline: none;
-    }
-  }
-
-  label,
-  .sage-checkbox__message {
-    color: var(--invalid-color);
-  }
-}
-
 .sage-checkbox--disabled {
```

Comment:
Looks like we'll need to clean this one up too

Reply:
Refactored and removed completely. 👍🏼